airbyte-webapp/src/components/ArrayOfObjectsEditor/components/EditorRow.module.scss

```diff
@@ -0,0 +1,42 @@
+@use "../../../scss/colors";
+@use "../../../scss/variables";
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  color: colors.$dark-blue;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 17px;
+  padding: variables.$spacing-md 8px;
+  border-bottom: 1px solid colors.$white;
```

Comment:
:art: Suggestion:

```css
.container + .container {
  bottom-top: 1px solid colors.$white;
}
```

So we can avoid setting and "unsetting" in `:last-child` again.